services/task-svc/migrations/000009_init.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE patients
+	ADD is_discharged INTEGER NOT NULL DEFAULT 0;
```

Comment:
```suggestion
	ADD is_discharged BOOLEAN NOT NULL DEFAULT 0;
```
I mean, `INTEGER` also works ...

Reply:
error: migration failed: column "is_discharged" is of type boolean but default expression is of type integer in line 0: ALTER TABLE patients
        ADD is_discharged BOOLEAN NOT NULL DEFAULT 0;
 (details: pq: column "is_discharged" is of type boolean but default expression is of type integer)